### PR TITLE
Add integration tests of RoleRepository

### PR DESCRIPTION
**Pull request by Jok33rr**
Check it pls

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,6 +58,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+        <dependency>
```
Comment (Bellkross) [inline pom.xml]: Why did you add this dependency?

File: src/main/java/com/smarthouse/user/controller/UserController.java [hunk 2]
```diff
@@ -26,14 +26,14 @@ public UserController(final UserServiceImpl userService, final DeviceService dev
     }
 
     @GetMapping("/user/{id}")
-    public ResponseEntity<User> getUserById(@PathVariable("id") int id) {
+    public ResponseEntity<User> getUserById(@PathVariable("id") long id) {
```
Comment (Bellkross) [inline src/main/java/com/smarthouse/user/controller/UserController.java]: Why have you changed UserController.java class?

File: src/main/java/com/smarthouse/user/entity/Role.java [hunk 3]
```diff
@@ -14,23 +13,23 @@
 public class Role {
 
     @Id
-    @GeneratedValue(strategy= GenerationType.IDENTITY)
-    private int id;
+    @GeneratedValue
```
Comment (Bellkross) [inline src/main/java/com/smarthouse/user/entity/Role.java]: The same question: For which reason have you made these changes?

**Review by borsch — COMMENTED** [review]
there is no need to test standard `Spring Data` repository

**Review by Bellkross — COMMENTED** [review]
> there is no need to test standard `Spring Data` repository

OK
We do this simple and useless tests just for "calibrating" our communication in GitHub and Yaroslav's project understanding.

File: pom.xml (hunk 1 above)
Comment (Jok33rr) [reply pom.xml]: This dependency is redundant. Is is already deleted.
